lib/MailQueueHandler.php

```diff
@@ -434,7 +434,7 @@ protected function parseEvent($lang, IEvent $event) {
 		foreach ($this->activityManager->getProviders() as $provider) {
 			try {
 				$event = $provider->parse($lang, $event);
-			} catch (\InvalidArgumentException $e) {
+			} catch (\Throwable $e) {
```

Comment:
Should we log the exception?
Can you give some explanation for the change?

Reply:
https://github.com/nextcloud/deck/pull/5533 + https://github.com/nextcloud/deck/pull/5542

But yeah also not a fan of it and I think we should not do this.

Reply:
Added logging

Reply:
The reason for this change is that a single error in some app can block the sending of all activity emails.